### ISSUE TITLE
oioproxy: NSINFO reload error Decoding error with 17.04 conscience

### ISSUE THREAD

**Issue by jkasarherou** [open-io/oio-sds#1218]
using an oioproxy in 16.10 release (3.2.X) and connecting to a 17.04 release (4.X) conscience we have:

```
79244920 39 BADC log WRN o.p NSINFO reload error [OPENIO] from [172.18.0.2:6000]: (500) (code=500) - Decoding error (code=500) - invalid data
```

**Comment by jkasarherou** [comment]
Incomplete NS info
```
openio --oio-ns OPENIO cluster show
+-----------+--------+
| Field     | Value  |
+-----------+--------+
| namespace | OPENIO |
| chunksize | 1      |
+-----------+--------+
```

**Comment by jkasarherou** [comment]
Note that the oioproxy can still be used to create objects/etc in this state. I find this behavior undesirable.

**Comment by jfsmig** [comment]
The best answer possible for the smoothest migration possible as been merged in branches 4.1.x, 4.x and master
